extensions/cornerstone-dicom-rt/src/viewports/OHIFCornerstoneRTViewport.tsx

```diff
@@ -91,12 +97,31 @@ function OHIFCornerstoneRTViewport(props: withAppTypes) {
     });
   }, [viewportGrid]);
 
-  const hydrateRTDisplaySet = ({ rtDisplaySet, viewportId }) => {
-    commandsManager.runCommand('loadSegmentationDisplaySetsForViewport', {
-      displaySets: [rtDisplaySet],
-      viewportId,
-    });
-  };
+  const hydrateRTDisplaySet = useCallback(
```

Comment:
Should this full function be a commands manager run command?  It feels like hydrating an RT display set might be done via a command/external function.

Reply:
Done